app/js/moduleView/moduleViewControllers.js

```diff
@@ -410,7 +410,7 @@ manageModuleController.controller('ModuleListCtrl',
                 }).success(function (data, status, headers, config) {
                     $scope.isStartAllModules = false;
                     $scope.getAllModuleDetails();
-                    $scope.startAllModuleSuccess = "Action completed successfully. Please check the module's status";
+                    $scope.startAllModuleSuccess = "Action completed successfully. Please check the modules' status";
```

Comment:
Or would adding just the word "each" make it easier to understand?
```suggestion
                    $scope.startAllModuleSuccess = "Action completed successfully. Please check each module's status";
```

Reply:
Yes, It will add more value to the message. @JyothsnaAshok Can you change it accordingly?